Unix/deprecated/httpclientcxx/httpclientcxx.cpp

```diff
@@ -530,7 +530,7 @@ void IOThread::StartRequestTh(NotifyItem* item)
         c_headers.data = &headers_pointers[0];
     }
 
-    headers_pointers.push_back("Authorization: None");
+    headers_pointers.insert(headers_pointers.begin(), "Authorization: None");
```

Comment:
It does matter, due to some convoluted reasons and the fact that httpclientcxx.cpp is deprecated and doesn't use the new httpclient interface. Definitely want this in the beginning, so that CM can override it by setting its own Authorization header in the extraHeaders.